### PR TITLE
Navegacion

### PR DESCRIPTION
**Pull request by lmaroso**
# Navegación

## Summary

Adding Navigation to Todo App.

## Task functionalities

- _Navigation between two screens:._
     - _Todo._
     - _DummyScreen._

## Changes:

- _Refactor of Todo App as Santi required on last PR._

#### Todo Screen

Screenshot_20190731-121622

#### Dummy Screen.

Screenshot_20190731-121631

### CODE REVIEW

File: src/components/Title/index.js [hunk 1]
```diff
@@ -4,10 +4,9 @@ import { Text, View } from 'react-native';
 import styles from './styles';
 
 const Title = () => {
-  const { textStyle, viewStyle } = styles;
   return (
-    <View style={viewStyle}>
-      <Text style={textStyle}>Todo List</Text>
+    <View style={styles.view}>
+      <Text style={styles.text}>Todo List</Text>
```
Comment (sdandois) [inline src/components/Title/index.js:9]: Move this string to a constant.

File: src/components/Todo/styles.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 const styles = StyleSheet.create({
-  footerStyle: {
+  footer: {
     position: 'absolute',
     left: 0,
     right: 0,
     bottom: 0
+  },
+  viewStyle: {
```
Comment (sdandois) [inline src/components/Todo/styles.js:10]: remove `Style` suffix.

File: src/redux/todo/reducer.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ function* createIds() {
 }
 const generateId = createIds();
 
-export default (state = INITIAL_STATE, action) => {
+export default function reducer(state = INITIAL_STATE, action) {
```
Comment (sdandois) [inline src/redux/todo/reducer.js:11]: Why the change? Use the arrow function instead of using the functon keyword.

File: src/Router/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    navigationOptions: ({ navigation }) => ({
+      title: `${navigation.state.routes[navigation.state.index].routeName}`
+    })
+  }
+);
+
+const stackNavigator = createStackNavigator(
+  {
+    Home: tabNavigator
```
Comment (sdandois) [inline src/Router/index.js:36]: Home is actually  a route name. You should added to the Routes object and use it like this `[Routes.Home]: `.

**Review by sdandois — CHANGES_REQUESTED** [review]
Great work! We should do same extra refactoring related to navigation.

1. Create a directory named `src/screens/`
2. Move Todo and Dummy Screen There.

**EDIT** This will be done in futer pr

File: src/components/Title/index.js (hunk 1 above)
Comment (lmaroso) [reply src/components/Title/index.js:9]: I've moved both screens to "src/screens" on a new branch called "bookList".
About the constant requested, it's now managed by router's "navigationOption" configuration.

File: src/components/Todo/styles.js (hunk 2 above)
Comment (lmaroso) [reply src/components/Todo/styles.js:10]: Done on another branch called "bookList".

File: src/redux/todo/reducer.js (hunk 3 above)
Comment (lmaroso) [reply src/redux/todo/reducer.js:11]: Done on another branch called "bookList".

File: src/Router/index.js (hunk 4 above)
Comment (lmaroso) [reply src/Router/index.js:36]: Done on another branch called "bookList".

**Review by sdandois — APPROVED** [review]
